pipeline-notebooks/pipeline-general.ipynb

```diff
@@ -685,6 +705,11 @@
     "            detail=f\"Invalid strategy: {strategy}. Must be one of {strategies}\"\n",
     "        )\n",
     "        \n",
+    "    # Note (austin) - there is no fast mode for images\n",
+    "    # In this case we need to use hi_res\n",
+    "    if file_content_type == \"image/jpeg\":\n",
+    "        strategy = \"hi_res\"\n",
```

Comment:
this should probably be handled at the unstructured `partition` level

Reply:
i can bump unstructured again, but when i was testing the ocr parameters i noticed `hi_res` does not work with `ocr_languages` parameter. so maybe we should covert strategy to `auto` if a `fast` strategy is passed in for images in unstructured?

Reply:
i think `fast` -> `ocr_only` for images makes sense, but this should happen in the unstructured lib.

> i noticed hi_res does not work with ocr_languages parameter. 

er, is this a bug for unstructured itself? seems like it should.

Reply:
l i think its expected: https://github.com/Unstructured-IO/unstructured/blob/main/unstructured/partition/strategies.py#L127-L137

Reply:
For now I'll disable the failing test on the api side